spec/controllers/admin/csv_export_spec.rb

```diff
@@ -25,12 +25,14 @@
 
       expect(user["first_name"]).to eq(@person.given_name)
       expect(user["last_name"]).to eq(@person.family_name)
+      expect(user["display_name"]).to eq(@person.display_name || "")
```

Comment:
My thinking went so that if you are to add to FactoryGirl a display_name it'll still work as intended. Currently, display_name is nil and then it's supposed to be empty string.